src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -2475,14 +2471,14 @@ When nodes are removed from the slave cluster, or if nodes go down or come back
 
 ==== Keeping Track of Logs
 
-Each master cluster region server has its own znode in the replication znodes hierarchy.
-It contains one znode per peer cluster (if 5 slave clusters, 5 znodes are created), and each of these contain a queue of WALs to process.
+Before 3.0.0, for zookeeper based implementation, it is like a tree, we have a znode for a peer cluster, but under the znode we have lots of files.
```

Comment:
I think here we'd better make two different sections to describe the logic before and after 3.0.0. As on zookeeper, we store all WAL files on it and for table based solution, we only store an offset.